.github/CODEOWNERS

```diff
@@ -1,40 +1,40 @@
 # Each line is a file pattern followed by one or more owners.
 # See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
 /src/analysis @DataBiosphere/manifold-analysis
-/src/billing-core/ @DataBiosphere/broad-core-services
-/src/billing/ @DataBiosphere/broad-core-services
-/src/groups/ @DataBiosphere/broad-core-services
+/src/billing-core/ @DataBiosphere/manifold-core-services
+/src/billing/ @DataBiosphere/manifold-core-services
+/src/groups/ @DataBiosphere/manifold-core-services
 /src/pages/EnvironmentsPage/ @DataBiosphere/manifold-analysis
 /src/constants/datasets.js @DataBiosphere/jadeteam
 /src/images/brands/scientificServices/ @DataBiosphere/teaspoons-developers
-/src/import-data/ @DataBiosphere/jadeteam @DataBiosphere/broad-core-services
+/src/import-data/ @DataBiosphere/jadeteam @DataBiosphere/manifold-core-services
```

Comment:
I think this, `/src/workspace-data/`, and `/integration-tests/utils/import-tdr-snapshot.ts` probably make sense to go to Data Management, but that can happen whenever they take care of their code owners changes

Reply:
This should have been included in my PR: https://github.com/DataBiosphere/terra-ui/pull/5469